bin/generate_core_lockfile.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+
+require 'bundler'
+ENV["BUNDLE_GEMFILE"] = "/tmp/lock_generator/Gemfile"
+
+#Set up working directory
+require 'fileutils'
+FileUtils.mkdir_p("/tmp/lock_generator/bundler.d")
+FileUtils.cp("/mnt/Gemfile", "/tmp/lock_generator/")
+FileUtils.cp("/mnt/Gemfile.release.rb", "/tmp/lock_generator/bundler.d/") if File.file?("/mnt/Gemfile.release.rb")
+branch = Bundler.load.dependencies.detect { |i| i.name == "manageiq-api" }.branch
+puts "Bundling for branch: #{branch}..."
+exit $?.exitstatus unless system("git clone https://github.com/ManageIQ/manageiq-appliance.git -b #{branch} /tmp/manageiq-appliance")
```

Comment:
You might want to FuleUtils.rm_rf("/tmp/manageiq-appliance") before this otherwise git might complain if the dir already exists.

Reply:
There shouldn't be anything in `/tmp`.